io/disk/parallel_dd.py.data/parallel_dd.yaml

```diff
@@ -0,0 +1,23 @@
+setup:
+    filesystem: !mux
+        default:
+            disk:
+    mb: !mux
+        default:
+            megabytes: 100
+    blocks: !mux
+        first:
+            blocks:
```

Comment:
can you please pass some default value of all these option , i face hard time to pass the value as it is string or integer  or you an add this README  but i prefer yaml